docs/Authenticating-Users-via-LDAP.md

```diff
@@ -48,11 +46,7 @@ Please note that you will have to modify all the above to match your own setting
 
 ## Authorizing Users
 
-Within portal.properties, modify
-
-    authorization=true
-
-Next, please read the Wiki page on [User Authorization](User-Authorization.md), and add user rights for a single user.
+Please read the Wiki page on [User Authorization](User-Authorization.md), and add user rights for a single user.
```

Comment:
Here maybe we should add a note that user authorizations only apply to portal deployments which require Authentication, and are not necessary for portals that use authentication=false or authentication=social_auth